tests/commands_test.py

```diff
@@ -30,8 +31,9 @@ def test_command(arg: List[str]) -> int:
 
         shell = TestShell(commands=[test_command])
         self.assertEqual(22, shell.run_cli_line("test_shell test-command --arg a b"))
-        self.assertEqual(22, shell.run_interactive_line('test-command arg=["a","b"]'))
-        self.assertEqual(22, shell.run_interactive_line("test-command arg=[a, b]"))
+        loop = asyncio.get_event_loop()
+        self.assertEqual(22, loop.run_until_complete(shell.run_interactive_line('test-command arg=["a","b"]')))
```

Comment:
Since you are migrating tests to async, having `loop.run_until_complete` adds too much noise to the code. How about we use something like `https://docs.python.org/3/library/unittest.html#unittest.IsolatedAsyncioTestCase` and to support Python < 3.8, we can use "later" https://github.com/facebookincubator/later to pull.

What do you think?